package.json

```diff
@@ -48,9 +47,12 @@
     "eslint-config-prettier": "^3.1.0",
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-prettier": "^2.6.2",
+    "markdown-table": "^1.1.3",
     "mocha": "^5.2.0",
     "mocha-lcov-reporter": "1.3.0",
-    "nyc": "^13.0.1"
+    "nyc": "^13.0.1",
+    "semver": "^6.3.0",
```

Comment:
`semver` is used by one of the rules (`compiler-version`) so it can't be a dev dependency.